src/engine/QueryExecutionTree.cpp

```diff
@@ -166,6 +166,9 @@ std::shared_ptr<QueryExecutionTree> QueryExecutionTree::createSortedTree(
 
   // Unwrap sort to avoid stacking sorts on top of each other.
   if (auto sort = std::dynamic_pointer_cast<Sort>(qet->getRootOperation())) {
+    AD_LOG_WARN << "Tried to re-sort a subtree that will already be sorted "
+                   "with `Sort` with a different sort order. This is a bug."
+                << std::endl;
```

Comment:
This is not a bug per se but
`The reason is possibly a missed optimization in the query planner`.

Reply:
So please rephrase this, so we don't get that many complaints of innocent users.